downstream/assemblies/platform/assembly-ug-controller-job-templates.adoc

```diff
@@ -22,7 +22,7 @@ Search functionality for Job templates is limited to alphanumeric characters onl
 
 //image::ug-job-templates-home.png[Job templates home]
 
-From this screen you can launch image:rightrocket.png[Rightrocket,15,15], edit image:leftpencil.png[Leftpencil,15,15], duplicate image:copy.png[Copy,15,15] and delete image:delete-button.png[Delete,15.15] a job template.
+From the template card you can launch image:rightrocket.png[Rightrocket,15,15], edit image:leftpencil.png[Leftpencil,15,15] a template, or, using the {MoreActionsIcon} icon, you can duplicate image:copy.png[Duplicate,15,15] or delete image:delete-button.png[Delete,15,15]a template.
```

Comment:
Looks like you might need a space after "image:delete-button.png[Delete,15,15]" (before "a template".